src/Lib/Lionk.Plugin/Model/Plugin.cs

```diff
@@ -52,7 +60,7 @@ public Plugin(Assembly assembly)
     /// <summary>
     /// Gets the dependencies of the plugin as an array of strings.
     /// </summary>
-    public string[] Dependencies { get; }
+    public List<Dependency> Dependencies { get; }
```

Comment:
This method only has a get, and it's built with the constructior. Could it be a readonly collection?

Reply:
Public properties with getter can't be readonly unfortunately :(